src/Elmish.WPF/Binding.fs

```diff
@@ -94,20 +96,32 @@ module Binding =
     |> mapData (alterMsgStream alteration)
 
 
+  /// <summary>
+  ///   Strongly-typed bindings that update the view from the model.
+  ///   Can be used in binding lists if boxed using <see cref="boxT" />.
```

Comment:
Remove this line for non-SubModelT modules. You'd never actually do this (just remove the `T` from the module)